zerver/webhooks/statuspage/view.py

```diff
@@ -0,0 +1,40 @@
+# Webhooks for external integrations.
+
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.response import json_success
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import get_client, UserProfile
+from django.http import HttpRequest, HttpResponse
+from typing import Dict, Any, Text
+
+
+@api_key_only_webhook_view('Statuspage')
+@has_request_variables
+def api_statuspage_webhook(request: HttpRequest, user_profile: UserProfile,
+                           payload: Dict[str, Any]=REQ(argument_type='body'),
+                           stream: str=REQ(default='statuspage-test'),
+                           topic: str=REQ(default='Statuspage')) -> HttpResponse:
+
+    status = payload["page"]["status_description"]
```

Comment:
I would recommend using `payload["page"]["status_indicator"]`. Relying on the description to figure out what kind of event a payload corresponds to can be wrong. Descriptions can be anything and can change. :)